src/System.Data.SqlClient/tests/System.Data.SqlClient.Tests.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>$(NetMinimum);$(NetMinimum)-unix;$(NetMinimum)-windows;$(NetFrameworkMinimum)</TargetFrameworks>

```

Comment:
Probably you need something to filter to the subset that you want to test - can be done separately.

Reply:
Do you mean `dotnet test System.Data.SqlClient.tests /p:TargetFramework=net6.0-unix`? Because it works - the dll is generated and it is also found when running the tests (even though there are none):

```
dotnet test .\System.Data.SqlClient.Tests.csproj /p:TargetFramework=net6.0-unix 
  Determining projects to restore...
  Restored C:\Users\calope\source\repos\maintenance-packages\src\System.Data.SqlClient\tests\System.Data.SqlClient.Tests.csproj (in 562 ms).
  1 of 2 projects are up-to-date for restore.
  System.Data.SqlClient -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Data.SqlClient\Debug\net6.0-unix\System.Data.SqlClient.dll
  System.Data.SqlClient.Tests -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Data.SqlClient.Tests\Debug\net6.0-unix\System.Data.SqlClient.Tests.dll
Test run for C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Data.SqlClient.Tests\Debug\net6.0-unix\System.Data.SqlClient.Tests.dll (.NETCoreApp,Version=v6.0)
Microsoft (R) Test Execution Command Line Tool Version 17.9.0 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
No test is available in C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Data.SqlClient.Tests\Debug\net6.0-unix\System.Data.SqlClient.Tests.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
Results File: C:\Users\calope\source\repos\maintenance-packages\artifacts\TestResults\Debug\System.Data.SqlClient.Tests_net6.0-unix_x64.trx
Html test results file : C:\Users\calope\source\repos\maintenance-packages\artifacts\TestResults\Debug\System.Data.SqlClient.Tests_net6.0-unix_x64.html
```